modules/smart-agent_logstash/conf/01-events_in_high.yaml

```diff
@@ -0,0 +1,23 @@
+module: logstash
+name: events in high
+
+transformation: ".mean(over='10m')"
+aggregation: true
+
+signals:
+  signal:
+    metric: node.stats.events.events.in
+    rollup: delta
+rules:
+  major:
+    description: is high
+    threshold: 25000
+    comparator: '>='
+    dependency: critical
+    lasting_duration: 10m
+  critical:
```

Comment:
`critical`/`major` seem to be a little too high as severity. I would use `minor`/`warning` instead because it is relative to performance issue and not availability: https://github.com/claranet/terraform-signalfx-detectors/wiki/Notifications-binding#minor.

Reply:
I'll make change to this suggestion.